executor/common_linux.h

```diff
@@ -1357,6 +1357,117 @@ static long syz_emit_ethernet(volatile long a0, volatile long a1, volatile long
 }
 #endif
 
+#if SYZ_EXECUTOR || __NR__syz_io_uring_put_sqes_on_ring
+//#include <linux/fs.h> // for __kernel_rwf_t
+
+// from liburing (tools/io_uring/barrier.h)
```

Comment:
Please use the following comment style:

```
// From liburing (tools/io_uring/barrier.h).
```

Here and below.

Reply:
Fixed.